README.md

```diff
@@ -41,7 +41,7 @@ A Collections of programming [Cheat Sheets](./CheatSheets/) for developers to qu
 9.  [python-cheatsheet.md](./CheatSheets/Python-cheatsheet.md)
 10. [html-cheatsheet.md](./CheatSheets/html-cheatsheet.md)
 11. [css-cheatsheet.md](./CheatSheets/css-cheatsheet.md)
-
+12. [java-cheatsheet.md](./CheatSheets/java-cheatsheet.md)
```

Comment:
This line of code created a `conflict`. Please, make your branch up to date, then link in the `README.md` file.

Before running the `git pull` command to make your branch up-to-date, please, copy your `java-cheatsheet.md` file. Otherwise, It will be gone.